src/main/cpp/libmatrixmult.h

```diff
@@ -17,48 +17,19 @@
  * under the License.
  */
 
-#ifndef _libmatrixmult_h
-#define _libmatrixmult_h
+#ifndef LIBMATRIXMULT_H
+#define LIBMATRIXMULT_H
```

Comment:
is there a convention for this, all CAPS?

Reply:
Depending on your conventions - yes. I was/is a convention for most people I've worked with. I must have adopted it at some point.